### DISCUSSION TITLE
Docker Django code example - composeexample

### DISCUSSION THREAD

**Discussion by eddhyne** [community/community#64093] [Programming Help]
### Select Topic Area

Question

### Body

I am running awsome-compose/offical-documentation-samples/django and I can’t get 
docker compose run web django-admin startproject composeexample .  to work I get
PS Z:\Docker+Django\Docker download from Docker Hub\Ed's Code\code\EdsDjangoApp> docker compose run web django-admin startproject composeexample .
[+] Creating 1/0
 ✔ Container edsdjangoapp-db-1  Running                                                                                                                                                                              0.0s 
CommandError: /code/manage.py already exists. Overlaying a project into an existing directory won't replace conflicting files.
When I run it without the period at the end I get:
PS Z:\Docker+Django\Docker download from Docker Hub\Ed's Code\code\EdsDjangoApp> docker compose run web django-admin startproject composeexample
[+] Creating 1/0
 ✔ Container edsdjangoapp-db-1  Created                                                                                                                                                                              0.0s 
[+] Running 1/1
 ✔ Container edsdjangoapp-db-1  Started                                                                                                                                                                              0.5s 
CommandError: '/code/composeexample' already exists

When I do DIR, I don’t see it
Directory: Z:\Docker+Django\Docker download from Docker Hub\Ed's Code\code\EdsDjangoApp

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----         8/16/2023   6:10 AM                data
-a----         8/16/2023   6:04 AM            498 docker-compose.yml
-a----         8/16/2023   6:03 AM            195 Dockerfile

What am I doing wrong?
-a----         8/16/2023   6:03 AM             31 requirements.txt

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @eddhyne ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by eddhyne** [comment]
Where do I find that category. Please give me a link.

[redacted due to PII]

**Reply by queenofcorgis** [reply]
Your post is already moved and the link will remain the same. You can find the Programming Category here: https://github.com/orgs/community/discussions/categories/programming-help

**Comment by eddhyne** [comment]
Thanks very much.

Ed Dhyne
Direct: 954-608-6603
***@***.******@***.***>
www.linkedin.com

**Comment by eddhyne** [comment]
![Snag_480810](https://github.com/community/community/assets/92999884/cef13c30-5fc6-4f15-a814-a21637a99096)

**Comment by eddhyne** [comment]
I got it to run but it does not create the composeexample directory. Please help.
[NoComposeExample.docx](https://github.com/community/community/files/12386138/NoComposeExample.docx)

**Comment by hamzasial1911** [comment]
update your Dockerfile 
add this CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]
in the end of the Dockerfile 
after adding that run the containers (docker-compose build --up)
docker run -it --name yourcontainer 
this will open the bash of docker
now execute the command 
```django-admin startproject myproject```
